comp120_1507866.tex

```diff
@@ -0,0 +1,54 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Report}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1507866}
+
+\begin{document}
+	
+\maketitle
+	
+\abstract{This report will look over the last semester and reflect on what I believe to have been my three main weaknesses}.
+	
+\section{Introduction}
+After reviewing my weekly reports for this semester I feel that my three main weaknesses are my time management, problems solving and critiquing skills.
+
+\section{Time Management}
+Time management is the skill that needs the most improvement. I have found when I have many assignments set it can be overwhelming. It would be better to plan and assign time to each one.
+
```

Comment:
You could merge these fragmented sentences into a single paragraph within each section. Avoid short paragraphs.